src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/integration/IntegrationTestBase.kt

```diff
@@ -127,4 +134,16 @@ abstract class IntegrationTestBase {
     httpHeaders.set(HttpHeaders.AUTHORIZATION, "Bearer $token")
     return httpHeaders
   }
+
+  fun aDefendant(personId: UUID = UUID.randomUUID(), givenName: String, familyName: String, pncIdentifier: PNCIdentifier = PNCIdentifier.from("2001/0171310W"), crn: String = "CRN1234", dateOfBirth: LocalDate = LocalDate.of(1965, 6, 18)) {
+    val person = Person(otherIdentifiers = OtherIdentifiers(pncIdentifier = pncIdentifier, crn = crn), givenName = givenName, familyName = familyName, dateOfBirth = dateOfBirth)
+
+    val newPersonEntity = PersonEntity(personId = personId)
+    newPersonEntity.createdBy = "test"
+    newPersonEntity.lastUpdatedBy = "test"
+    val newDefendantEntity = DefendantEntity.from(person)
+    newDefendantEntity.person = newPersonEntity
+    newPersonEntity.defendants.add(newDefendantEntity)
+    personRepository.saveAndFlush(newPersonEntity)
+  }
 }
```

Comment:
This class is getting bigger. I think we need to start thinking about moving the common data to separate files.